src/Form/Form.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+
+class Form extends Component {
+  constructor() {
+    super()
+    this.state = {
+      searchTerm: ''
+    }
+  }
+
+  // searchMoviesHere = () => {
```

Comment:
Remember to remove this one - we solved the problem by passing the same data into setState and into searchMovies, to make them run synchronously.